### PR TITLE
[DI][FrameworkBundle] Remove "whitelist" occurrences

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
  */
 class UnusedTagsPass implements CompilerPassInterface
 {
-    private $whitelist = [
+    private $excludeList = [
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php:24]: this reverses the meaning of the word

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php:24]: allowlist?

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php]: changed for allowlist

**Review by javiereguiluz — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php [hunk 2]
```diff
@@ -21,7 +21,7 @@
  */
 class UnusedTagsPass implements CompilerPassInterface
 {
-    private $whitelist = [
+    private $allowlist = [
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php:24]: knownTags?

Comment (fabpot) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php:24]: I like Nicolas' suggestion.

Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/UnusedTagsPass.php]: 👍  updated

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**